mappings/net/minecraft/datafixer/fix/ArrowPickupFix.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_5946 net/minecraft/datafixer/fix/ArrowPickupFix
+	COMMENT A fix that automatically converts the {@code player} NBT field in arrows
+	COMMENT to {@code pickup} field if possible.
```

Comment:
```suggestion
	COMMENT A fix that automatically renames the {@code player} byte in arrow data to
	COMMENT {@code pickup}, if there is not any existing {@code pickup} data.
```

Reply:
Sure, though I don't see the difference

Reply:
I think yarn should avoid referring to the contents of NBT compounds as fields

Reply:
Ah. Now I've fixed the other reference of NBT compound element as "field" in javadoc, mind review again?